test/e2e/lib/components/site-editor-component.js

```diff
@@ -156,4 +194,32 @@ export default class SiteEditorComponent extends AsyncBaseContainer {
 			)
 		);
 	}
+
+	async dismissNotices() {
+		const locator = By.css( '.components-snackbar[aria-label="Dismiss this notice"]' );
+		const notices = await this.driver.findElements( locator );
+		await Promise.all( notices.map( ( notice ) => notice.click() ) );
```

Comment:
Let's use the dedicated helper and make sure each next clicks is awaited:
```suggestion
		for ( const notice of notices ) {
				await driverHelper.clickWhenClickable( this.driver, () => notice );
		}
```

Reply:
`TIL` Passing a function as a locator